SpywareFilter/sections/specific.txt

```diff
@@ -42,6 +42,7 @@
 !
 /analytics.js$domain=bataryapil.com|journey.com.tr|tozlu.com|elleshoes.com|freexcafe.com|rusvideos.tv|nakubani.ru|lun.com|songsara.net|partifabrik.com|newsmax.com
 !
+||analytics.clic2buy.com/c2b-analytics.js
```

Comment:
It seems that this rule can be made short by committing the following suggestion:
```suggestion
||analytics.clic2buy.com^
```